features/step_definitions/networking.rb

```diff
@@ -975,3 +981,23 @@
   raise "Failed to kill the #{ovndb} database daemon" unless @result[:success]
 end
 
+
+# work-around nested clipboard Transform <% cb.south_leader.name %> issues by combining this step
+Given /^admin deletes the ovnkube-master#{OPT_QUOTED} leader$/ do |ovndb|
+  ensure_admin_tagged
+
+  cb_leader_name ||= "#{ovndb}_leader"
+  if cb[cb_leader_name] == nil
+    step %Q/I store the ovnkube-master "#{ovndb}" leader pod in the :#{cb_leader_name} clipboard/
+  end
+  leader_pod_name = cb[cb_leader_name].name
+  # this doens't work for some reason, can't find the dynamic step
+  # step %Q/Given admin ensures "#{leader_pod_name}" pod is deleted from the "openshift-ovn-kubernetes" project/
+
+  _resource = resource(leader_pod_name, "pod", project_name: "openshift-ovn-kubernetes")
+  p = proc {
+    @result = _resource.ensure_deleted(user: admin, wait: 300)
+  }
+  p.call
```

Comment:
if this step is still needed after my suggestion about expression evaluation above, you can do here just
```
@result = resource(leader_pod_name, "pod", project_name: "openshift-ovn-kubernetes").ensure_deleted(user: admin, wait: 300)
```

Reply:
done